.github/workflows/publish_dockerhub_images.yml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Build And Publish DockerHub images
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_VERSION:
+        type: string
+        description: Release Version
+        required: true
+        default: "2.42.X"
+      RC_NUM:
+        type: string
+        description: Release Candidate Number
+        required: true
+        default: "1"
+      FINAL_PUBLISH:
+        type: boolean
+        description: Publish as RELEASE_VERSION and LATEST?
+        required: true
+        default: false
+
+jobs:
+  build:
+    env:
+      RELEASE_VERSION: ${{github.event.inputs.RELEASE_VERSION}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{secrets.DOCKER_HUB_REPO}}
+          password: ${{secrets.DOCKER_HUB_TOKEN}}
+      - name: Setting Release Branch
+        run: echo "RC_TAG=v${{env.RELEASE_VERSION}}-RC${{github.event.inputs.RC_NUM}}" >> $GITHUB_ENV    
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{env.RC_TAG}}
+      - name: Setup Go 
+        uses: actions/setup-go@v3
+        with:
+          go-version: '^1.13.1' 
```

Comment:
This seems odd - why are we (a) starting with such a low version (b) pinning to the minor version, vs [using the standard action](https://github.com/apache/beam/blob/master/.github/actions/setup-self-hosted-action/action.yml) for grabbing our current supported versions

Reply:
Java 17 wasn't required, I'm just leaving Java 8 and 11, and Go version 1.19.